### PR TITLE
Fix #396 cascade and use GITHUB_TOKEN

### LINKED ISSUE #396: 20230109 cascade and use GITHUB_TOKEN

### PR DESCRIPTION
**Pull request by robnagler**
There's really no way to test this without hacking build scripts. This needs to be reviewed for errors, manually.

### CODE REVIEW

File: installers/python-ci/radiasoft-download.sh [hunk 1]
```diff
@@ -28,8 +28,9 @@ python_ci_main() {
             i=radiasoft/sirepo
             ;;
     esac
-    local d=$PWD
-    local o=$(stat --format='%u:%g' "$d")
+    declare d=$PWD
+    declare o=$(stat --format='%u:%g' "$d")
+    declare c=
```
Comment (rorour) [inline installers/python-ci/radiasoft-download.sh:33]: What is c?

Comment (robnagler) [reply installers/python-ci/radiasoft-download.sh]: 19cf1327

File: installers/python-ci/radiasoft-download.sh [hunk 2]
```diff
@@ -6,9 +6,9 @@ python_ci_main() {
     if [[ ! -r setup.py ]]; then
         install_err 'no setup.py'
     fi
-    local i
-    local p=0
-    local r=$(basename "${GITHUB_REPOSITORY:-MISSING}")
+    declare i
+    declare p=0
+    declare r=$(basename "${GITHUB_REPOSITORY:-MISSING}")
```
Comment (rorour) [inline installers/python-ci/radiasoft-download.sh:11]: Why change these from local to declare?

**Review by rorour — APPROVED** [review]

File: installers/python-ci/radiasoft-download.sh (hunk 2 above)
Comment (robnagler) [reply installers/python-ci/radiasoft-download.sh:11]: https://github.com/radiasoft/discuss/issues/15

Comment (robnagler) [reply installers/python-ci/radiasoft-download.sh:11]: Resolve privately.

**Pull request merged**